dropshot/tests/test_demo.rs

```diff
@@ -606,6 +609,35 @@ async fn test_delete_request() {
     testctx.teardown().await;
 }
 
+/*
+ * Test response header
+ */
+#[tokio::test]
+async fn test_header_request() {
+    let api = demo_api();
+    let testctx = common::test_setup("test_header_request", api);
+    let response = testctx
+        .client_testctx
+        .make_request(
+            Method::GET,
+            "/testing/header",
+            None as Option<()>,
+            StatusCode::NO_CONTENT,
+        )
+        .await
+        .expect("expected success");
+
+    println!("{:?}", response.headers());
+
+    let header = response
```

Comment:
What do you think of adding another header to this test case so that we test what happens if you add the header multiple times?  (That's regardless of what we decide to do in that case.)

Reply:
Good idea.